src/System.IO.FileSystem/src/System/IO/FileStatus.Unix.cs

```diff
@@ -113,6 +113,20 @@ public void SetAttributes(string path, FileAttributes attributes)
             if (!_exists)
                 FileSystemInfo.ThrowNotFound(path);
 
+            if (Interop.Sys.CanSetHiddenFlag)
+            {
+                if ((attributes & FileAttributes.Hidden) != 0 && (_fileStatus.UserFlags & (uint)Interop.Sys.UserFlags.UF_HIDDEN) != (uint)Interop.Sys.UserFlags.UF_HIDDEN)
+                {
+                    // If Hidden flag is set and cached file status does not have the flag set then set it
+                    Interop.Sys.LChflags(path, (uint)Interop.Sys.UserFlags.UF_HIDDEN);
```

Comment:
Is it ok that we're ignoring errors?  Assuming yes, a comment to that effect (saying why) would be good.

Reply:
I checked the Windows implementation of `SetAttributes` and it throws when the operation returned != 0 code

https://github.com/dotnet/corefx/blob/77096755443730962810662501f82550fdb4656d/src/System.IO.FileSystem/src/System/IO/FileSystem.Windows.cs#L569-L575

also Unix one throws

https://github.com/dotnet/corefx/blob/4fd3d39f831f3e14f311b0cdc0a33d662e684a9c/src/System.IO.FileSystem/src/System/IO/FileStatus.Unix.cs#L147

I will wrap it in the `Interop.CheckIo` to match the existing unix implementation which uses `InteropChMod`.

Reply:
Sounds good.